react/features/base/color-scheme/defaultScheme.js

```diff
@@ -47,8 +47,8 @@ export default {
         background: 'rgb(249, 249, 249)'
     },
     'Thumbnail': {
-        activeParticipantHighlight: 'rgb(81, 214, 170)',
-        activeParticipantTint: 'rgba(49, 183, 106, 0.3)',
+        activeParticipantHighlight: 'rgb(243, 146, 160)',
+        activeParticipantTint: 'rgb(243, 146, 160)', // cabi pink?
```

Comment:
Why the question mark in the comment? Also, could this be extracted to a constant somewhere? This is a pretty common color, used in foreseeably in many places.